public/src/js/models/config/persistence.js

```diff
@@ -65,7 +68,8 @@ function flattenModel (model) {
     return _.reduce(model, function (accumulator, value, key) {
         var x = flattenValue(value);
 
-        if (x) {
+        // We want to drop falsy values, but not 0
```

Comment:
Cf. https://github.com/guardian/facia-tool/pull/682
> This pr also cleans all falsy values and empty arrays from the meta object to correspond with what is saved in the meta by v1 and avoid any obscure future bugs.

There are a couple of places in the tests where we are explicitly testing falsy values are filtered out, so this seemed the minimum change to allow us to support 0 as a value.

0 as a value is useful as it lets config editors know that a group (e.g. Very Big) has explicitly been set to 0 rather than it is unset and waiting to be filled in.